src/runtime_src/core/tools/xbmgmt2/SubCmdProgram.cpp

```diff
@@ -148,16 +183,11 @@ update_SC(unsigned int  index, const std::string& file)
 
   auto dev = xrt_core::get_mgmtpf_device(index);
 
-  //versal flow to flash sc
-  try {
-    uint32_t val = xrt_core::query::program_sc::value_type(1);
-    xrt_core::device_update<xrt_core::query::program_sc>(dev.get(), val);
+  bool is_versal = xrt_core::device_query<xrt_core::query::is_versal>(dev);
```

Comment:
This Query is not defined in Windows. It throws an exception for Alveo devices as well. 
You need to define and return false as Windows doesnt support versal yet.

Reply:
Added query in windows as well, thanks @chvamshi-xilinx